manager/frontend/src/containers/Feed.js

```diff
@@ -54,7 +55,12 @@ export default function Feed() {
                     <div className={postClasses.join(' ')}>
                         <PostCreator />
                         <PostSorter />
-                        <Posts posts={temp_posts} />
+                        {simplifiedPosts.map( postData =>
```

Comment:
shouldn't this be passed to Posts

Reply:
postData is the post data for one post. simplfiedPosts is an array of postData